addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -379,7 +379,11 @@ const threadPatch = {
     },
     /** @param {string} body */
     async post(body) {
-        if (this.model === "discuss.channel" && body.startsWith("/")) {
+        if (
+            this.model === "discuss.channel" &&
+            body.startsWith("/") &&
+            this.store.self.type !== "guest"
```

Comment:
A small addition to the remark about patches: the condition is not exactly the same, meaning that after this change, guests will be able to select a command from the public page (the `channel_type === "livechat"` condition does not match, thus allowing the command delimiter), but the command won't be considered during post (since the channel type is not checked).